README.md

```diff
@@ -224,6 +225,27 @@ It is recommended that you keep your PDS up to date with new versions, otherwise
 sudo pdsadmin update
 ```
 
+### Environment Variables
+
+| Environment Variable | Value | Default |
+|---------------------|-------|---------|
+| `PDS_HOSTNAME` | Empty | None |
+| `PDS_JWT_SECRET` | Empty | None |
+| `PDS_ADMIN_PASSWORD` | Empty | None |
+| `PDS_PLC_ROTATION_KEY_K256_PRIVATE_KEY_HEX` | Empty | None |
+| `PDS_DATA_DIRECTORY` | `/pds` | `/pds` |
+| `PDS_BLOBSTORE_DISK_LOCATION` | `/pds/blocks` | `/pds/blocks` |
+| `PDS_BLOB_UPLOAD_LIMIT` | `52428800` | `52428800` (50MB) |
```

Comment:
I think we should actually bump this up to `104857600`. We increased the limit on the bsky-hosted PDS fleet a while back. Also in `sample.env`.

In the longer term we might want to leave this un-set in the template, and have the default value set in PDS codebase. Then the configured value would only apply if a PDS operator actually set one.

Reply:
oh, and update the `installer.sh` itself! I guess that could all be a separate PR from this one.